atd-vze/src/queries/gqlAbstract.js

```diff
@@ -224,10 +244,20 @@ gqlAbstractTableAggregateName (
 
     if (this.config["where"] !== null) {
       let where = [];
+      let _or = [];
```

Comment:
I don't have a strong opinion about `_` prefix for indicating private/class variables and methods in JS. But it seems like we should at least be consistent with the existing `where` variable naming. Either both underscore or neither. Unless I'm misunderstanding.

Reply:
ohhh, wait, it has to do with the naming of the `_or` query param in the graphql query object, right? my bad.

Reply:
Yep, it is following the GraphQL syntax but it feels a bit out of place here. I changed it everywhere except for where the gql query is built since that makes two of us. Tested and it works.